sql/src/main/java/io/crate/analyze/relations/DocTableRelation.java

```diff
@@ -90,9 +87,6 @@ public Field getField(Path path) {
     @Override
     public Field getField(Path path, Operation operation) throws UnsupportedOperationException, ColumnUnknownException {
         ColumnIdent ci = toColumnIdent(path);
-        if (HIDDEN_COLUMNS.contains(ci)) {
```

Comment:
It's necessary to be able to resolve the _fetchId  here: https://github.com/crate/crate/pull/5493/files#diff-630bac56f4217582a384868fccb29c64R77